PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -172,10 +178,12 @@ internal class PayPalInternalClient(
         }
     }
 
-    private fun createAppSwitchUri(uri: Uri): Uri {
+    private fun createAppSwitchUri(uri: Uri, flowType: String, merchantId: String): Uri {
```

Comment:
It might be good to add a Kdoc and explain why `flowType` and `merchantId` are added to the app switch Uri.